products/almalinux.md

```diff
@@ -41,12 +46,27 @@ releases:
 > [AlmaLinux OS](https://almalinux.org/) is an open source, community-owned and governed,
 > forever-free enterprise Linux distribution, focused on long-term stability, providing a robust
 > production-grade platform. AlmaLinux OS is binary-compatible with
-> [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux), and is owned
-> and controlled by the non-profit AlmaLinux OS Foundation, managed by a community-elected
-> board of directors.
+> [RHEL](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux). It is owned
+> and controlled by the non-profit AlmaLinux OS Foundation, and managed by a community-elected
+> board of directors and the self-managed AlmaLinux Engineering Steering Committee.
 
-AlmaLinux OS attempts to release regular updates within 1 business day from RHEL. Each release, like
+AlmaLinux OS attempts to release regular updates within 1 business day of RHEL. Each release, like
 RHEL, comes with a 10-year lifecycle. Major releases are typically available within a few days to a
 few weeks following a major RHEL release.
 
+## Improvements and differences from RHEL
+### Lifecycles 
+Unlike RHEL, each minor version reaches end of life when the new version is released. For example, AlmaLinux OS 9.2 reached end of life with the release of AlmaLinux OS 9.3.
+
+### Feature additions
+In Jun 2023 Red Hat announced they would no longer be providing the source code that AlmaLinux and others used to build RHEL-like OS's for public consumption without creation of a user account. The user agreement indicates that Red Hat reserves the right to terminate a relationship with anyone who uses Red Hat's code to create a distribution. AlmaLinux has agreed to build their OS in the way that Red Hat requested. As a result of that change, the project has also been able to adopt differentiations from Red Hat to better serve its community. Those differences are outlined in each version's release notes, but primarily include extending hardware support that RHEL has removed, adding architecture support beyond what Red Hat provides, and enabling support for things like KVM and SPICE.
```

Comment:
```
In June 2023, Red Hat announced that they would no longer be providing the source code that AlmaLinux and others used to build RHEL-like operating systems for public consumption without the creation of a user account. The user agreement indicates that Red Hat reserves the right to terminate a relationship with anyone who uses Red Hat's code to create a distribution. AlmaLinux has agreed to build their operating system in the way that Red Hat requested. As a result of that change, the project has also been able to adopt differentiations from Red Hat to better serve its community. Those differences are outlined in each version's release notes but primarily include extending hardware support that RHEL has removed, adding architecture support beyond what Red Hat provides, and enabling support for technologies like KVM and SPICE.
```

Reply:
@usta I don't understand what you're suggesting here. Can you please use suggestions instead of comments? that will make it more clear what you're specifically looking for. https://haacked.com/archive/2019/06/03/suggested-changes/